pkg/ocmagenthandler/ocmagenthandler_pdb.go

```diff
@@ -29,7 +29,8 @@ func buildOCMAgentPodDisruptionBudget(ocmAgent ocmagentv1alpha1.OcmAgent) *v1.Po
 			},
 			Selector: &metav1.LabelSelector{
 				MatchLabels: map[string]string{
-					"app": "ocm-agent",
```

Comment:
I think we just need to update this line to match the same in deployment
```
"app": ocmAgent.Name,
```
why we need to keep the `app=ocm-agent` and add an additional label `ocmagent-name`?

Reply:
My bad, I was going dynamically set an additional label using ocmAgent.Name , but it's already set in the deployment and we just need to match the label in PDB